### ISSUE TITLE
V7 logout is not idempotent

### ISSUE THREAD

**Issue by lisamburns** [cloudfoundry/cli#2011]
- [x] I reviewed open and closed github issues that may be related to my problem.
- [x] I tried updating to the latest version of the CF CLI to see if it fixed my problem.
- [x] I attempted to run the command with `CF_TRACE=1` to help debug the issue.
- [x] I am reporting a bug that others will be able to reproduce.
- [x] If this is an issue for the v7 beta release, I've read through the [official docs](https://docs.cloudfoundry.org/cf-cli/v7.html) and the [release notes](https://github.com/cloudfoundry/cli/releases).

**Describe the bug and the command you saw an issue with**
cf logout command fails if you aren't targeted to an api.
This is irritating when you want to write scripts that ensure that any pre-existing logins are cleaned up first.
The cf cli version 6 is idempotent and does not have this issue.

**What happened**
cf logout exits 1

**Expected behavior**
cf logout exits 0

**To Reproduce**
Steps to reproduce the behavior; include the exact CLI commands and verbose output:
1. Run `rm -rf ~/.cf`to make sure there is no api targeted.
2. Run `cf logout`
3. See error
```
➜  relint-workstation git:(master) ✗ ~/Downloads/cf7-cli_7.0.2_osx/cf7 logout -v
No API endpoint set. Use 'cf7 login' or 'cf7 api' to target an endpoint.
FAILED
```

**Provide more context**
Mac OS X
CLI v7.0.2

**Comment by heyjcollins** [comment]
Thanks for pointing out this regression @lisamburns !
We've prioritized fixing this in our backlog and will apply the fix to both the v6 and v7 lines.

**Comment by heyjcollins** [comment]
this issue has been fixed in the v6 CLI v6.53.0

**Comment by wayneadams** [comment]
@heyjcollins , it looks like the fix is not yet in v7. The latest version is [v7.1.0](https://github.com/cloudfoundry/cli/releases/tag/v7.1.0). Is there v7.2.0 forthcoming?
